### ISSUE TITLE
device_class: water is not recognized yet as a valid entry

### ISSUE THREAD

**Issue by sammyke007** [hassio-addons/addon-vscode#538]
# Problem/Motivation

**device_class: water** gives an "error" in Studio Code Server

## Expected behavior

Accept water as a valid entry

## Actual behavior

Red flag

## Steps to reproduce

/

## Proposed changes

Merge https://github.com/keesschollaart81/vscode-home-assistant/pull/2349

![image](https://user-images.githubusercontent.com/33524412/206421835-791584ac-c676-40eb-92fd-52359bd70dbd.png)

**Comment by frenck** [comment]
As you linked in the proposed change, that is a PR in another project.
Closing the issue here, as it is not an issue of the add-on but upstream.

../Frenck

**Comment by sammyke007** [comment]
Hi Frenck. Can you ELI5 please? 😅
